### PR TITLE
grpc_check: Fix timeout default value

### PR DESCRIPTION
**Pull request by keisku**
### What does this PR do?

Set 1000 in timeout as default.

### Motivation

Same motivation as https://github.com/DataDog/integrations-extras/pull/1574, but different approach.

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Comment by keisku** [comment]
The [CI job](https://github.com/DataDog/integrations-extras/actions/runs/3424901279/jobs/5705127360) failed and suggested running `ddev validate config grpc_check -s`.
However, running `ddev validate config grpc_check -s` in my local looks fine.

Any advise to fix this CI failure?

**Comment by FlorentClarret** [comment]
> The [CI job](https://github.com/DataDog/integrations-extras/actions/runs/3424901279/jobs/5705127360) failed and suggested running `ddev validate config grpc_check -s`. However, running `ddev validate config grpc_check -s` in my local looks fine.
> 
> ```shell
> $ git status
> On branch keisku/fix-timeout-default
> nothing to commit, working tree clean
> $ git --no-pager show --summary
> commit fe48170b38c4474217795146b47058bb1b55fe62 (HEAD -> keisku/fix-timeout-default, origin/keisku/fix-timeout-default)
> Author: keisku <keisuke.umegaki@datadoghq.com>
> Date:   Wed Nov 9 11:38:57 2022 +0900
> 
>     fix timeout default value
> $ pwd
> /Users/keisukeumegaki/DDrepos/integrations-extras
> $ ddev validate config grpc_check -s
> Validating default configuration files for 1 checks...
> All 2 configuration files are valid!
> ```
> 
> Any advise to fix this CI failure?

You probably have an outdated version of ddev. You will need to upgrade. I pushed the modification.

Also, in order to get this released you will have to update the changelog, [the version number](https://github.com/DataDog/integrations-extras/blob/master/grpc_check/datadog_checks/grpc_check/__about__.py) and the instructions in the README. The release will be triggered automatically once merged

File: grpc_check/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
 # CHANGELOG - gRPC Check
 
+## 1.0.2 / 2022-11-09
+[Fixed] Fix timeout default value. See #1601.
```
Comment (FlorentClarret) [inline grpc_check/CHANGELOG.md:4]: nit 

```suggestion
[Fixed] Fix timeout default value. See [#1601](https://github.com/DataDog/integrations-extras/pull/1601).
```

**Review by maycmlee — APPROVED** [review]
👍 for docs

**Pull request merged**